sdk/storage/Azure.Storage.Blobs/src/BlobClientOptions.cs

```diff
@@ -31,7 +31,7 @@ public enum ServiceVersion
             /// The 2018-11-09 service version described at
             /// <see href="https://docs.microsoft.com/en-us/rest/api/storageservices/version-2018-11-09" />
             /// </summary>
-            V2018_11_09 = 0
+            V2018_11_09 = 1
```

Comment:
While we're in here making changes, can we please change all of these from `V2018_11_09` to `V2019_02_02`?  We've decided we're going to use that version for Blobs/Files/Queues across all our languages.  Please update the comments and link to https://docs.microsoft.com/en-us/rest/api/storageservices/versioning-for-the-azure-storage-services#version-2019-02-02 as well.